src/components/filters/useOperatingSystemFilter.js

```diff
@@ -34,7 +34,11 @@ const useOperatingSystemFilter = (apiParams = []) => {
   const [groups, setGroups] = useState([]);
 
   useEffect(() => {
-    dispatch(fetchOperatingSystems(apiParams));
+    if (typeof hasAccess === 'undefined') {
+      dispatch(fetchOperatingSystems(apiParams));
+    } else if (typeof hasAccess !== 'undefined' && hasAccess === true) {
```

Comment:
This may look weird. I know. However, this is in order to keep compatibility for now with places that use the hook but do not provide a `hasAccess` argument.